### DISCUSSION TITLE
Sync Vrealize Operation to Netbox

### DISCUSSION THREAD

**Discussion by ngocnd1** [bb-Ricardo/netbox-sync#424] [Ideas]
Hi anyone,

Recently get driectly from vcenter is litte bit risk, recommend from security department
Is there any disscussion about sync object from Vrealize Operation to Netbox?

Thanks

**Comment by bb-Ricardo** [comment]
Hi,

No, there is currently no implementation planned.

Could not find an API which gives an example of the objects returned by the API.

Also suspecting more and more people will e moving away from VMWare due to the recent price issue.
